main.nf

```diff
@@ -1474,6 +1474,7 @@ process SPADES {
                                                                  ch_spades_blast,
                                                                  ch_spades_abacas,
                                                                  ch_spades_plasmidid
+    set val(sample), val(single_end), file("*scaffolds.fa"), file("*assembly.gfa") into ch_spades_graph

```

Comment:
```suggestion
    file "*assembly.gfa" into ch_spades_graph

```

Reply:
Doesnt look like these files are being used in any downstream processes yet so better to not give them a named channel 👍